test/tag-test.js

```diff
@@ -8,15 +8,35 @@ suite('plugin-test', () => {
     return code.split(splitter).filter((token) => Boolean(token));
   }
 
-  test('it can transform queries with the gql tag', () => {
-    const result = transform('gql(client)`{shop{name}}`;', {plugins: ['./src/index.js']});
+  test('it can transform queries tagged with the imported gql function', () => {
+    const result = transform("import {gql} from './src/index';\ngql(client)`{shop{name}}`;", {plugins: ['./src/index.js']});
 
     assert.deepEqual(tokens(result.code), tokens(`
+      import { gql } from './src/index';
```

Comment:
Babel inserts the spaces when transforming which messes up the tokenizing if they're not in the expected result as well.